src/main/java/com/kkm/kkm_server_v2/global/infra/S3/service/AwsS3Service.java

```diff
@@ -41,15 +42,15 @@ public List<String> uploadFile (List<MultipartFile> multipartFile) {
                 amazonS3.putObject(new PutObjectRequest(bucket, fileName, inputStream, objectMetadata)
                         .withCannedAcl(CannedAccessControlList.PublicRead));
             } catch (IOException e) {
-                throw new ResponseStatusException(HttpStatus.INTERNAL_SERVER_ERROR, "파일 업로드에 실패했습니다.");
+                throw FileUploadFailedException.EXCEPTION;
             }
-            fileNameList.add("https://kkm.s3.ap-northeast-2.amazonaws.com/"+fileName);
+            fileNameList.add(address+fileName);
         });
```

Comment:
forEach 구문이 아닌 map 구문을 사용하여 .collect(Collectors.toList()); 로 fileNameList를 반환하게 만드는 것이 어떨까요?

```suggestion
List<String> fileNameList = multipartFile.map(item -> {
대충 s3 코드
return address+fileName;
}).collect(Collectors.toList());
```